### ISSUE TITLE
ThemeProvider broken in IE10.

### ISSUE THREAD

**Issue by pleunv** [styled-components/styled-components#355]
## Version
1.2.1

## Reproduction
No WebpackBin as not IE10 compatible. Put up a quick repro here: https://github.com/pleunv/styled-components-themeprovider-ie10.

See https://github.com/pleunv/styled-components-themeprovider-ie10/blob/master/app/containers/HomePage/index.js and https://github.com/pleunv/styled-components-themeprovider-ie10/blob/master/app/components/Nav/Nav.js for code.

```js
const Nav = styled.div`
  background-color: ${(props) => props.bgColor || props.theme.bgColor || 'red'};
`;
```

## Steps to reproduce
Run project, open up http://localhost:3000 in IE10 (or IE11 with IE10 emulation)

## Expected Behavior

Theme variables to be passed in the props.

![Chrome](http://i.imgur.com/BjNVATi.png)

## Actual Behavior

Theme object is passed in but it contains no properties. As a result the default value is being used. No errors are being thrown.

![IE10](http://i.imgur.com/Fy2s2O6.png)
![IE DevTools](http://i.imgur.com/sNRodOA.png)

**Comment by mxstbr** [comment]
Do we have somebody on Windows who can dig into this?

**Comment by saschagehlich** [comment]
Just stumbled over this bug. Will investigate. Yay.

**Comment by saschagehlich** [comment]
@mxstbr Found it - this is the problem: https://babeljs.io/docs/usage/caveats/#internet-explorer-classes-10-and-below-

This causes `StyledComponent` to not copy `ParentComponent`'s `contextTypes` property, so the `__styled-components__` context property is not available. A possible workaround is to add `static contextTypes = ParentComponent.contextTypes` to the class.

**Comment by mxstbr** [comment]
Fixed by the magnificent @saschagehlich in #379. Thanks so much!

Will release a new version asap.
